samples/extensions/extended_dynamic_state2/README.md

```diff
@@ -0,0 +1,387 @@
+	vertex_input_state.vertexAttributeDescriptionCount      = static_cast<uint32_t>(vertex_input_attributes.size());
+	vertex_input_state.pVertexAttributeDescriptions         = vertex_input_attributes.data();
+
+	std::array<VkPipelineShaderStageCreateInfo, 4> shader_stages{};
+	shader_stages[0] = load_shader("extended_dynamic_state2/baseline.vert", VK_SHADER_STAGE_VERTEX_BIT);
+	shader_stages[1] = load_shader("extended_dynamic_state2/baseline.frag", VK_SHADER_STAGE_FRAGMENT_BIT);
+
+	/* Create graphics pipeline for dynamic rendering */
+	VkFormat color_rendering_format = render_context->get_format();
+
+	/* Provide information for dynamic rendering */
+	VkPipelineRenderingCreateInfoKHR pipeline_create{VK_STRUCTURE_TYPE_PIPELINE_RENDERING_CREATE_INFO_KHR};
+	pipeline_create.pNext                   = VK_NULL_HANDLE;
+	pipeline_create.colorAttachmentCount    = 1;
+	pipeline_create.pColorAttachmentFormats = &color_rendering_format;
+	pipeline_create.depthAttachmentFormat   = depth_format;
+	pipeline_create.stencilAttachmentFormat = depth_format;
+
+	/* Skybox pipeline (background cube) */
+	VkSpecializationInfo                    specialization_info;
+	std::array<VkSpecializationMapEntry, 1> specialization_map_entries{};
+	specialization_map_entries[0]        = vkb::initializers::specialization_map_entry(0, 0, sizeof(uint32_t));
+	uint32_t shadertype                  = 0;
+	specialization_info                  = vkb::initializers::specialization_info(1, specialization_map_entries.data(), sizeof(shadertype), &shadertype);
+	shader_stages[0].pSpecializationInfo = &specialization_info;
+	shader_stages[1].pSpecializationInfo = &specialization_info;
+
+	/* Use the pNext to point to the rendering create struct */
+	VkGraphicsPipelineCreateInfo graphics_create{VK_STRUCTURE_TYPE_GRAPHICS_PIPELINE_CREATE_INFO};
+	graphics_create.pNext               = VK_NULL_HANDLE;
+	graphics_create.renderPass          = VK_NULL_HANDLE;
+	graphics_create.pInputAssemblyState = &input_assembly_state;
+	graphics_create.pRasterizationState = &rasterization_state;
+	graphics_create.pColorBlendState    = &color_blend_state;
+	graphics_create.pMultisampleState   = &multisample_state;
+	graphics_create.pViewportState      = &viewport_state;
+	graphics_create.pDepthStencilState  = &depth_stencil_state;
+	graphics_create.pDynamicState       = &dynamic_state;
+	graphics_create.pVertexInputState   = &vertex_input_state;
+	graphics_create.pTessellationState  = VK_NULL_HANDLE;
+	graphics_create.stageCount          = 2;
+	graphics_create.pStages             = shader_stages.data();
+	graphics_create.layout              = pipeline_layouts.baseline;
+
+	
+
+	graphics_create.pNext      = VK_NULL_HANDLE;
+	graphics_create.renderPass = render_pass;
+
+	VK_CHECK(vkCreateGraphicsPipelines(get_device().get_handle(), pipeline_cache, 1, &graphics_create, VK_NULL_HANDLE, &pipeline.baseline));
+...
+```
+
+And now, thanks to dynamic state 2, we can change parameters before each corresponding draw call.
+
```

Comment:
Describe that in draw_from_scene we have an implementation of depth bias and rasterizer discard

Reply:
Done added code snippet of draw_from_scene  function.